recipes/gdist/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: gdist
+  version: "1.0.3"
+
+source:
+  md5: 98547eac8e7c0b5a7a600e12ae5c9c4c
+  fn: 'gdist-1.0.3.tar.gz'
+  url: https://pypi.io/packages/source/g/gdist/gdist-1.0.3.tar.gz
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [py>27]
```

Comment:
Does it actually not work with python 3? Looking at the package, I don't see any reason why it wouldn't.